src/test/java/seedu/address/model/person/DepartmentContainsKeywordsPredicateTest.java

```diff
@@ -0,0 +1,94 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.testutil.PersonBuilder;
+
+public class DepartmentContainsKeywordsPredicateTest {
+
+    @Test
+    public void equals() {
+        List<String> firstPredicateKeywordList = Collections.singletonList("first");
+        List<String> secondPredicateKeywordList = Arrays.asList("first", "second");
+
+        DepartmentContainsKeywordsPredicate firstPredicate =
+                new DepartmentContainsKeywordsPredicate(firstPredicateKeywordList);
+        DepartmentContainsKeywordsPredicate secondPredicate =
+                new DepartmentContainsKeywordsPredicate(secondPredicateKeywordList);
+
+        // same object -> returns true
+        assertTrue(firstPredicate.equals(firstPredicate));
+
+        // same values -> returns true
+        DepartmentContainsKeywordsPredicate firstPredicateCopy =
+                new DepartmentContainsKeywordsPredicate(firstPredicateKeywordList);
+        assertTrue(firstPredicate.equals(firstPredicateCopy));
+
+        // different types -> returns false
+        assertFalse(firstPredicate.equals(1));
+
+        // null -> returns false
+        assertFalse(firstPredicate.equals(null));
```

Comment:
I like the comprehensive test cases but perhaps you could use more more edge case test cases like null and empty strings here for all the other test cases?

Reply:
What other test cases?

Reply:
Every other test case without null tests